content/blog/entries/gsoc2019-arimadian/contents.lr

```diff
@@ -0,0 +1,28 @@
+title: Ari Madian - My GSoC 2019 Blog
+---
+categories:
+GSoC
+Google Summer of Code
+cc-chooser
+cc-license-chooser
+---
+author: akmadian
```

Comment:
please add author page per step 4 here: https://creativecommons.github.io/community/write-a-blog-post/

Reply:
the lack of an author page causes a build failure as you can see in https://travis-ci.org/creativecommons/creativecommons.github.io-source/builds/544503455, please run the build locally before you submit the PR (it's in the PR template checklist!)